packages/editor/src/components/default-block-appender/index.js

```diff
@@ -67,11 +67,14 @@ export function DefaultBlockAppender( {
 				value={ showPrompt ? value : '' }
 			/>
 			<InserterWithShortcuts rootClientId={ rootClientId } layout={ layout } />
-			<Inserter position="top right">
-				<DotTip id="core/editor.inserter">
-					{ __( 'Welcome to the wonderful world of blocks! Click the “+” (“Add block”) button to add a new block. There are blocks available for all kind of content: you can insert text, headings, images, lists, and lots more!' ) }
-				</DotTip>
-			</Inserter>
+			<Inserter
+				position="top right"
+				dotTip={
```

Comment:
Can we make `Inserter` not aware of `DotTip` in a similar way?

Reply:
I tried, but if we do that the dotTip starts appearing in the wrong place. I think for the inserter case that is not problematic inserter is an editor component and not a general component.